Resources/public/js/views/cof-contentcreationwidgetview.js

```diff
@@ -0,0 +1,210 @@
+        SELECTOR_CONTENT_CREACTION = '.' + CLASS_CONTENT_CREACTION,
+        SELECTOR_PAGE_ONE = SELECTOR_CONTENT_CREACTION + '__page-one',
+        SELECTOR_PAGE_TWO = SELECTOR_CONTENT_CREACTION + '__page-two',
+        SELECTOR_NEXT_BUTTON = SELECTOR_CONTENT_CREACTION + '__pagination__btn--next',
+        SELECTOR_BACK_BUTTON = SELECTOR_CONTENT_CREACTION + '__pagination__btn--back',
+        SELECTOR_FINISH_BUTTON = SELECTOR_CONTENT_CREACTION + '__pagination__btn--finish',
+        SELECTOR_CONTENT_TYPE_SELECTOR =SELECTOR_CONTENT_CREACTION +  '__content-type-selector',
+        SELECTOR_CHANGE_CONTENT_TYPE = SELECTOR_CONTENT_CREACTION + '__btn--change-content-type',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_NEXT_BUTTON] = {'tap': '_changeFormPage'};
+    EVENTS[SELECTOR_BACK_BUTTON] = {'tap': '_changeFormPage'};
+    EVENTS[SELECTOR_CHANGE_CONTENT_TYPE] = {'tap': '_changeFormPage'};
+
+    /**
+     * The Content Creation Widget view
+     *
+     * @namespace cof
+     * @class ContentCreationWidgetView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.cof.ContentCreationWidgetView = Y.Base.create('ContentCreationWidgetView', Y.eZ.TemplateBasedView, [], {
+        events: EVENTS,
+
+        initializer: function () {
+            this.on('displayedChange', this._getContentTypes, this);
+            this.on('contentTypeGroupsChange', this._renderContentTypeSelector, this);
+            this.on('*:itemSelected', this._enableNextButton, this);
+        },
+
+        render: function () {
+            this.get('container').setHTML(this.template({
+                finishBtnText: this.get('finishBtnText')
+            }));
+
+            return this;
+        },
+
+        /**
+         * Enables the next button
+         *
+         * @protected
+         * @method _enableNextButton
+         * @param event {Object} event facade
+         */
+        _enableNextButton: function (event) {
+            var container = this.get('container');
+
+            container.one(SELECTOR_CHANGE_CONTENT_TYPE).setHTML(event.text);
+            container.one(SELECTOR_NEXT_BUTTON).removeClass(CLASS_BUTTON_DISABLED);
+        },
+
+        /**
+         * Gets the Content Types
+         *
+         * @protected
+         * @method _getContentTypes
+         * @param event {Object} event facade
+         */
+        _getContentTypes: function (event) {
+            var container = this.get('container');
+
+            if (!event.newVal) {
+                this._resetWidgetState();
+
+                return;
+            }
+
+            this.get('contentTypeSelectorView').hideTooltip();
+
+            container.one(SELECTOR_CONTENT_TYPE_SELECTOR).addClass(CLASS_SELECTOR_LOADING);
+            container.one(SELECTOR_NEXT_BUTTON).addClass(CLASS_BUTTON_DISABLED);
+
+            /**
+             * Fired to fetch the Content Types.
+             * Listened in the cof.Plugin.createContentSelectContentType
```

Comment:
`createContentSelectContentType` is a bit confusing name. I'm not sure what the plugin does, is it creating content or selecting a content type?